features/proposal/components/VoterDetailsCard/tests/VoterDetailsCard.spec.tsx

```diff
@@ -9,22 +9,6 @@ jest.mock('components/Icon', () => {
 });
 
 describe('VoterDetailsCard', () => {
-  // it('Should render component', () => {
```

Comment:
Please, fix it)

Reply:
I only remove unused, commented code from test :)